task/rhacs-deployment-check/3.71/README.md

```diff
@@ -2,48 +2,101 @@
+
+- **`rox_central_endpoint`**: The address:port tuple for StackRox Central. Default: **central.stackrox.svc:443**
+- **`insecure-skip-tls-verify`**: Skip verification the TLS certs of the Central endpoint and registry. Examples: _"true", **"false"**_.
+- **`rox_config_dir`**: Path to the roxctl config directory within the `roxctl-config` workspace (if machine to machine authentication is used). Mutually exclusive with **`rox_token_file`**. The path should be prefixed with `/roxctl-config`. Examples: _"/roxctl-config", **""**_.
+- **`rox_token_file`**: Path to the API Token file (if API Token authentication is used). Mutually exclusive with **`rox_config_dir`**. The path should be prefixed with `/rox-api-token-auth`. Examples: _**""**, "/rox-api-token-auth/rox_api_token"_.
+- `rox_image`: The image providing the roxctl tool (optional). Default: quay.io/stackrox-io/roxctl:4.4.2 (this is also the minimum version working with this task). 
+
+One of the **`rox_config_dir`** or **`rox_token_file`** parameter is required for the authentication against the remote Central to work.
+
 ## Workspaces
 
 - **source**: A [Workspace](https://github.com/tektoncd/pipeline/blob/main/docs/workspaces.md) containing the deployment manifest.
+- **roxctl-config**: An [optional workspace](https://github.com/tektoncd/pipeline/blob/main/docs/workspaces.md#optional-workspaces) containing the configuration for roxctl. Used to authenticate with the remote central using short-lived tokens. The content of this worksapce is ideally populated by a rhacs-m2m-authenticate TaskRun. This workspace is mutually exclusive with the `rox-api-token-auth` workspace.
+- **rox-api-token-auth**: An [optional workspace](https://github.com/tektoncd/pipeline/blob/main/docs/workspaces.md#optional-workspaces) containing a rox token file. Used to authenticate with the remote central. It is **strongly** recommended that this workspace be bound to a Kubernetes `Secret`. This workspace is mutually exclusive with the `roxctl-config` workspace.
 
 ## Usage
 
-Create secrets for authentication to RHACS Central endpoint and supply filesystem path to deployment manifest for checking.
+Configure machine to machine authentication or create secrets for authentication to RHACS Central endpoint and supply filesystem path to deployment manifest for checking.
+
+**Note:** This task requires at least Read permission on the Detection resource configured on RHACS Central side.
 
 Run this task after rhacs-image-scan to ensure most up to date CVE data for images referenced by the deployment.
 
 If the deployment violates one or more enforced policies, this task will return a failure and cause the pipeline run to fail. 
 
+### Configure using machine to machine authentication
+
+Check the [documentation](https://docs.openshift.com/acs/operating/manage-user-access/configure-short-lived-access.html#configure-short-lived-access_configure-short-lived-access) to configure the trust with the OIDC token issuer.
```

Comment:
Probably would be great to mention that it should be the underlying kubernetes clusters OIDC config we need to configure here where the tasks will be run - not sure how explicit we want to be here, I feel like this should be within the machine to machine tasks documentation and we just link to that here.